worlds/hylics2/__init__.py

```diff
@@ -37,7 +37,7 @@ class Hylics2World(World):
 
     topology_present: bool = True
 
-    data_version: 1
+    data_version = 1
```

Comment:
This change should probably be in a different pull request as a bug fix.
The only reason it's working without this fix is that it's getting the `1` value from the base class.

Reply:
Originally this set `data_version` to 0 as default to ensure new worlds were automatically in testing mode and that would have affected this world. It could be reverted now that I changed the default back to 1, but honestly it's such a small tweak that it's probably just fine in my opinion, but if you insist, I can revert it.

Reply:
Saying that it should be in a different PR is not saying that it shouldn't be in this PR.
It's not a problem if it's both here and in another PR.